### ISSUE TITLE
Buttons do not have focus states

### ISSUE THREAD

**Issue by AustinGil** [CKGrafico/papanasi#74]
**Describe the bug**
There is no way to tell when a button has focus. This is an accessibility and UX concern.

**To Reproduce**
Got to the buttons page and either click on one or tab to one. There is no focus state.

**Expected behavior**
Something should show you which button is focused.

**Comment by CKGrafico** [comment]
Thanks for your feedback!
Do you think will be enough if we just add the same as we do for :active?
![image](https://user-images.githubusercontent.com/2048511/209119063-990a8e8b-d86b-42db-b5c9-f365bf61ac09.png)

**Comment by AustinGil** [comment]
Yeah, I noticed that design treatment when I was holding the mouse button down, but it went away when I released, which was weird. 

I can't tell you just by looking, but the WCAG outlines color contrast requirements for compliance https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html.

Unfortunately, as a library author, it's hard to say for sure because it really depends on the implementation. If the blue button is on a blue background, you wouldn't see the focus ring, but that's on the developer. I would check the contrast ratio on a white (or almost white) background, and if you want to support dark mode, then also test against the default browser Canvas color (which I dont think is black).

There's some tools that allow you to compare color contrast between two colors, but I can tell you for sure that the ring around the white button will not pass WCAG requirements when placed on a white background. It's hard to see it even without visual impairments.

I know this was a lot. Hopefully it's more good than bad :)

**Comment by CKGrafico** [comment]
Thanks for the info, I understand what you say but papanasi is not responsible about the background, I mean this library is not a boostrap, we expect that the users will change the colors adapting css variables, then is OUR problem to create code that does something when :active but not to be sure that the background is going to work :)

I'll implement this asap thanks
